src/notifications/generate.ts

```diff
@@ -145,6 +147,19 @@ export const generateNotificationMap: Record<
       .descriptionComment(ctx.comment)
       .targetPost(ctx.post, ctx.comment)
       .avatarManyUsers([ctx.commenter]),
+  post_mention: (
+    builder,
+    ctx: NotificationPostContext & NotificationDoneByContext,
+  ) =>
+    builder
+      .referencePost(ctx.post)
+      .icon(NotificationIcon.Comment)
+      .description((ctx.post as WelcomePost).content)
+      .targetPost(ctx.post)
+      .setTargetUrlParameter([
```

Comment:
This is a generic post mention notification but seems like the parameters here are specific to welcome post